Parsers/ASimDns/Parsers/vimDnsMicrosoftSysmon.yaml

```diff
@@ -121,44 +121,6 @@ ParserQuery: |
       | where  (responsecodename=="*" or DnsResponseCodeName has responsecodename) // -- filter is not optimized
       // --
     };
-  let ParsedDnsEvent_WindowsEvent =(
```

Comment:
please keep these filters in separate lines:
 starttime:datetime=datetime(null), endtime:datetime=datetime(null)
    , srcipaddr:string='*'
    , domain_has_any:dynamic=dynamic([]) 
    , responsecodename:string='*', response_has_ipv4:string='*'
    , response_has_any_prefix:dynamic=dynamic([]) , eventtype:string='Query'
    , disabled:bool=false

Reply:
Done